language/fr.json

```diff
@@ -58,6 +58,10 @@
         {
           "label": "Mettre à l'échelle le texte afin de s'adapter au sein de la carte",
           "description": "Si cette option est décochée la taille de la carte sera adaptée au texte placé à l'intérieur"
+        },
+        {
+          "label": "Shuffle the Cards",
```

Comment:
OK, Tim, I'll have another look at my PR and see if I can use the existing 'shuffle function' and I'll also test the  'Save content state' feature, but I might need help for that.
As for the language files I do agree they should only be changed once a PR has been merged (and the translation process has been agreed on).
QUESTION: Considering that the new changes might be important, should I create a brand new branch, called e.g. "Branch_randomize_cards" and then the current Branch_shuffle_cards could be deleted?